### DISCUSSION TITLE
Make netbox_power_port_template.type & netbox_power_port.type freeform

### DISCUSSION THREAD

**Discussion by MinDBreaK** [netbox-community/ansible_modules#1047] [Ideas]
The current netbox version allows different power plugs that are not allowed by the module like : 
* `dc-terminal`
* `other`
* `hardwire`

To improve the future compatibility with netbox the field could be freeform or have a way to disable validation, which is welcomed but can hold back when a new type is introduced. 

I can open the issue with the suggestion that would be preferred.

**Comment by sc68cal** [comment]
If you create a PR and link to this discussion that works for me

**Reply by MinDBreaK** [reply]
Thx, the PR has been created. For now I only added missing types, but I might do a free-form as soon as I can. I basically exported options to JSON and added them in the options). I didn't want to change any behaviour in this one.

**Reply by sc68cal** [reply]
That works. Thank you for the contribution!
